torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -130,7 +130,8 @@ class FullyShardedDataParallel(nn.Module, _FSDPState):
     .. _`Xu et al.`: https://arxiv.org/abs/2004.13336
     .. _DeepSpeed: https://www.deepspeed.ai/
 
-    For advanced notes please refer to :ref:`fsdp_notes`.
+    To understand its advanced features, you can refer to the
```

Comment:
I think the `fsdp_notes` is meant to be for advanced users, discussing FSDP's internals, not about advanced features per se. It could be worth leaving the existing wording.